rllib/models/catalog.py

```diff
@@ -65,6 +65,9 @@
     "lstm_cell_size": 256,
     # Whether to feed a_{t-1}, r_{t-1} to LSTM.
     "lstm_use_prev_action_reward": False,
+    # Experimental (only works with `_use_trajectory_view_api`=True):
```

Comment:
Time-major models currently only work iff _use_trajectory_view_api is True, but it'll become the default once _use_trajectory_view_api becomes default as well.